GRAPHICS/SetParticleFxLoopedColour.md

```diff
@@ -8,15 +8,15 @@ ns: GRAPHICS
 void SET_PARTICLE_FX_LOOPED_COLOUR(int ptfxHandle, float r, float g, float b, BOOL p4);
 ```
 
-```
-only works on some fx's  
-p4 = 0  
-```
-
 ## Parameters
 * **ptfxHandle**: 
-* **r**: 
-* **g**: 
-* **b**: 
+* **r**: range from 0.0 to 1.0
+* **g**: range from 0.0 to 1.0
+* **b**: range from 0.0 to 1.0
 * **p4**: 
 
+
+only works on some fx's  
+p4 = 0
+
+You can use the [inverse lerp](https://www.gamedev.net/articles/programming/general-and-gameplay-programming/inverse-lerp-a-super-useful-yet-often-overlooked-function-r5230/) method to normalize in a range from 0.0 to 1.0 an rgb
```

Comment:
I just realized, all this needs to be above the the `## Parameters` section. It would also be nice if you clarified on which fx's this works on and elaborated on why `p4` needs to be false, rather than just `p4 = 0` 🙂